lib/eth.rb

```diff
@@ -25,20 +25,12 @@ def replayable_chain_id
       27
     end
 
-    def chain_id
-      configuration.chain_id
+    def default_chain_id
+      configuration.default_chain_id
     end
 
     def v_base
-      if chain_id
-        (chain_id * 2) + 35
-      else
-        replayable_chain_id
-      end
-    end
-
-    def prevent_replays?
-      !chain_id.nil?
+      replayable_chain_id
```

Comment:
We could do that; now that we keep the chain id in the tx object itself, all uses of v_base map to 27. I left it as is to avoid changing open_ssl.rb, and also because I thought it might be a nice mnemonic. But I can go either way; just let me know your preference.